lib/Sema/TypeCheckDecl.cpp

```diff
@@ -4212,6 +4067,169 @@ EmittedMembersRequest::evaluate(Evaluator &evaluator,
   return CD->getMembers();
 }
 
+Optional<OperatorDecl *> swift::FunctionOperatorRequest::getCachedResult() const {
```

Comment:
We've been defining getCachedResult()/cacheResult() in TypeCheckRequests.cpp instead, but in this case if you use internal caching you won't have to define them at all.